docs/ref/contrib/postgres/aggregates.txt

```diff
@@ -0,0 +1,235 @@
+=========================================
+PostgreSQL specific aggregation functions
+=========================================
+
+.. module:: django.contrib.postgres.aggregates
+   :synopsis: PosgreSQL specific aggregation functions
+
+All of these functions are available
+from the ``django.contrib.postgres.aggregates``.
+
+.. warning::
+
+    All functions come without default aliases,
```

Comment:
break line closer to 79 characters
I think warning is a bit strong for this. 'note' is probably fine